README.md

```diff
@@ -58,6 +58,17 @@ Client
 
 Go to localhost:8080 and you should have Citylines running in your machine.
 
+How to use Citylines with Docker
+======
+
+```
+docker-compose up --build
+docker-compose exec web rake db:migrate
```

Comment:
@alfredormz why don't you put the `rake db:migrate` as the command of a container that runs between `db` and `web`, inside the `docker-compose.yml`. So the web container depends_on the migrations container and the migrations container depends_on db.

Reply:
@alfredormz qué pensás de poner el yarn install en el Dockerfile ?